.github/workflows/ci.yml

```diff
@@ -191,6 +191,11 @@ jobs:
           gpg_private_key: ${{ secrets.CALYPTIA_GPG_KEY }}
           passphrase: ${{ secrets.CALYPTIA_GPG_KEY_PASSPHRASE }}
 
+      - name: Update the upx version
+        if: ${{ startsWith(github.ref, 'refs/tags/v') }}
```

Comment:
Why not always?

Reply:
> Why not always?

Because this is specific for goreleaser (used when releasing)